scripts/html_lint.py

```diff
@@ -106,11 +108,21 @@ def main():
         return 1
 
     exclude = [_DISABLE_MAP[d] for d in disable if d in _DISABLE_MAP]
-    clean_html = template_remover.clean(io.open(options['FILENAME']).read())
-    results = html_linter.lint(clean_html, exclude=exclude)
+
+    path = options['FILENAME']
+    is_file = os.path.isfile(path)
+    html_files = [path] if is_file else glob.glob('{}/*.html'.format(path))
```

Comment:
`html` is not the only extension that should be supported, `htm` and many template extensions should be supported as well.

Also, this is counterintuitive, as there is no recursion.

My opinion is that instead of supporting directories we could support multiple filenames.